golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -245,7 +245,7 @@ func makeTestKit(bank types.BankKeeper) (Keeper, sdk.Context) {
 	ms.MountStoreWithDB(vbankStoreKey, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(paramsStoreKey, sdk.StoreTypeIAVL, db)
 	ms.MountStoreWithDB(paramsTStoreKey, sdk.StoreTypeTransient, db)
-	ms.LoadLatestVersion()
+	_ = ms.LoadLatestVersion()
```

Comment:
```
golang/cosmos/x/vbank/vbank_test.go:248:22: Error return value of `ms.LoadLatestVersion` is not checked (errcheck)
	ms.LoadLatestVersion()
```

Reply:
As with the lien test, check this and panic if not nil.